stored_queries/GO/GO_get_ancestors.yaml

```diff
@@ -19,9 +19,18 @@ params:
         default: 0
         description: Result offset for pagination
         maximum: 100000
-query_prefix: WITH GO_test_term
+    ts:
+      type: integer
+      title: Versioning timestamp
+query_prefix: WITH GO_terms
 query: |
-  FOR v, e IN 1..1000000 OUTBOUND CONCAT("GO_test_term/", @key) GO_test_edges_isa
-      SORT v._key ASC
+  FOR t in GO_terms
+    FILTER t.id == @id
+    FILTER t.created <= @ts AND t.expired >= @ts
+    limit 1
+    FOR v, e IN 1..100 OUTBOUND t GO_edges
```

Comment:
Yeah so PRUNE is the inverted condition. You can do `PRUNE e.created > @ts OR e.expired < @ts` just like you say, and leave out the FILTER part entirely.

I think it makes sense to only traverse on edges that are current and stop, because we're only interested in the current, up to date graph of the data, and we don't want to mix  connections with old versions fo the graph.

Reply:
I tried
``` 
PRUNE e.created > @ts OR e.expired < @ts
```
It returned 0 result.
It is the same for 
```
PRUNE e.created <= @ts OR e.expired >= @ts
```
When I was using
```
FILTER e.created <= @ts AND e.expired >= @ts
```
It returned 16 results.